<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple Linear Regresion - Benetton Case</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MainWindow {
	background-color: #F0F0F0;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Data Set del Caso Benetton
(Sales &amp; Advertising)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gr√°fico</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Paso a Paso</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Valores ingresados</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTableWidget" name="ownValuesTable">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>204</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>5</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Sales
(Million
Euro)</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
       <property name="background">
        <color>
         <red>230</red>
         <green>230</green>
         <blue>230</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>178</green>
          <blue>238</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Advertising
(Million Euro)</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
       <property name="background">
        <color>
         <red>230</red>
         <green>230</green>
         <blue>230</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>178</green>
          <blue>238</blue>
         </color>
        </brush>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Rango: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="minRange">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>59</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string> - </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="maxRange">
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="datasetTable">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>325</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>9</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Sales
(Million
Euro) (Y)</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
       <property name="background">
        <color>
         <red>230</red>
         <green>230</green>
         <blue>230</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>205</green>
          <blue>50</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Advertising
(Million Euro) (X)</string>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
       <property name="background">
        <color>
         <red>230</red>
         <green>230</green>
         <blue>230</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>205</green>
          <blue>50</blue>
         </color>
        </brush>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>651</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>23</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>762</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>26</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>856</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>30</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>1,063</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>34</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>1,190</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string>43</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string>1,298</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string>48</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string>1,421</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="6" column="1">
       <property name="text">
        <string>52</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="7" column="0">
       <property name="text">
        <string>1,440</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="7" column="1">
       <property name="text">
        <string>57</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="8" column="0">
       <property name="text">
        <string>1,518</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
      <item row="8" column="1">
       <property name="text">
        <string>58</string>
       </property>
       <property name="textAlignment">
        <set>AlignTrailing|AlignVCenter</set>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="generateAdvertising">
      <property name="text">
       <string>Generar Advertising Aleatoriamente</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" rowspan="3">
     <widget class="QListWidget" name="stepByStepList">
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="midLineWidth">
       <number>7</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="plotHolder" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="resultsLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Cambria</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Œ≤‚ÇÅ = , Œ≤‚ÇÄ = </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Fuente: &lt;a href=&quot;https://www.displayr.com/what-is-linear-regression/&quot;&gt;https://www.displayr.com/what-is-linear-regression/&lt;/a&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextBrowserInteraction</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
